api/HTMLEmbedElement.json

```diff
@@ -52,6 +52,335 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "align": {
```

Comment:
Looks like content attribute entries are still missing for "align" and maybe other IDL attributes added here.